pokemongo_bot/cell_workers/handle_soft_ban.py

```diff
@@ -28,7 +28,7 @@ def work(self):
         )
 
         if fort_distance > Constants.MAX_DISTANCE_FORT_IS_REACHABLE:
-            MoveToFort(self.bot).work()
+            MoveToFort(self.bot, self.config).work()
```

Comment:
Agreed, I'm not sure how we fix this though. HandleSoftBan doesn't (and shouldn't) know what the config is for MoveToFort. Is there any way we can refactor HandleSoftBan to not require anything about MoveToFort? Or do we just say screw it create the config that MoveToFort wants (ignoring however the user might have configured that task) and use it?